summarize.go

```diff
@@ -326,3 +326,37 @@ func stringifySection(cfg Config, filter ValueFilterFunc, out *bytes.Buffer, s *
 		}
 	}
 }
+
+func cleanWhiteSpace(buffer *bytes.Buffer) string {
+	content := buffer.String()
+
+	lines := strings.Split(content, "\n")
+
+	// Process each line to trim leading and trailing whitespace
+	var processedLines []string
+	for _, line := range lines {
+		// Trim leading and trailing whitespace
+		trimmedLine := strings.TrimSpace(line)
+
+		// Preserve indentation by re-calculating indentation and re-adding it
+		if len(trimmedLine) > 0 {
+			// Calculate the leading spaces (indentation)
+			indentation := ""
+			for _, char := range line {
+				if char == ' ' {
+					indentation += " "
+				} else {
+					break
+				}
+			}
+
+			// Rebuild the line with preserved indentation but no trailing spaces
+			processedLines = append(processedLines, indentation+trimmedLine)
+		} else {
+			// Keep empty lines intact
+			processedLines = append(processedLines, "")
+		}
```

Comment:
I think instead of using TrimSpace, you can use TrimRight instead, which saves needing to do any logic on indenting

Reply:
```suggestion
		processedLines = append(processedLines, strings.TrimRight(line, " "))
```